lib/middleman-breadcrumbs/breadcrumbs.rb

```diff
@@ -21,7 +21,7 @@ def initialize(app, options_hash = {}, &block)
   def breadcrumbs(page, separator: @separator, wrapper: @wrapper)
     hierarchy = [page]
     hierarchy.unshift hierarchy.first.parent while hierarchy.first.parent
-    hierarchy.collect {|page| wrap link_to(page.data.title, "/#{page.path}"), wrapper: wrapper }.join(h separator)
+    hierarchy.collect { |x| wrap link_to(x.data.title, x.url), wrapper: wrapper }.join(h separator)
```

Comment:
Why are you changing the name of the block argument from `page` to `x`?

Reply:
Probably because `page` shadows the first `page` declared at the top of the method. Let's call it `p` instead of `x` maybe?

Reply:
To keep using the path instead of switching to the URL I'd do something like `URI(page.url).path` or `URI(x.url).path`. If that's alright I'll do a new pull request with an explanation of where the old way fails for me.